Dockerfile

```
@@ -31,4 +30,6 @@ EXPOSE 7170/tcp
 EXPOSE 7171/tcp
 
 WORKDIR /usr/src/monorepo/packages/broker
-CMD ./bin/broker.js # start broker from default config
+
+# start broker from default config
+CMD ["./bin/broker.js"]
```

```suggestion
ENTRYPOINT ["./bin/broker.js"]
```

I didn't try, but I think this should fix the issue where `init` is required.

---

@kare good thought, though I believe this will break the ability to use this image to start other executables

e.g. trackers in the docker-compose https://github.com/streamr-dev/streamr-docker-dev/blob/3c55bab982ed65d30722e65b36a0b0649298f0ce/docker-compose.yml#L128

something like this in the testnet instructions:
```
docker run -it -v $(cd ~/.streamrDocker; pwd):/root/.streamr \
  streamr/broker-node:testnet bin/config-wizard

```